mlxtend/plotting/plot_sequential_feature_selection.py

```diff
@@ -23,7 +23,7 @@ def plot_sequential_feature_selection(metric_dict,
     Parameters
     ----------
     metric_dict : mlxtend.SequentialFeatureSelector.get_metric_dict() object
-    figsize : tuple (default: (10,10))
+    figsize : tuple (default: (8,6))
```

Comment:
Based on the function definition above, this should be

    figsize : tuple (default: None)

I think choosing `None` is also reasonable as it would use the matplotlib defaults